amethyst_input/src/input_handler.rs

```diff
@@ -389,6 +393,43 @@ impl<T: BindingTypes> InputHandler<T> {
     pub fn send_frame_begin(&mut self) {
         self.mouse_wheel_vertical = 0.0;
         self.mouse_wheel_horizontal = 0.0;
+
+        // Prepare the mouse axis input
+        {
+            let hits_boundaries = |delta: f64| delta >= 1. || delta <= -1.;
+            let normalize = |x: f64| {
```

Comment:
Let's make this a private function rather than an in-line variable since we're using it multiple times.